src/protocol/resp/src/lib.rs

```diff
@@ -2,8 +2,11 @@
 // Licensed under the Apache License, Version 2.0
 // http://www.apache.org/licenses/LICENSE-2.0
 
-mod get;
-mod set;
+mod request;
```

Comment:
The basic types of RESP are used to represent both request and response.

I think the code structure should reflect this relationship, where `protocol/resp` should use one (sub-)module to represent primitive types and their serde operations; and the other module to represent commands and their serde, implemented on top of the primitive types / operations.

Reply:
It's true current redis commands are all multistrings, but I think if we want to expand it (e.g. allowing options/maps), we will make full use of all the types in both requests and responses.

Reply:
Makes sense. This sounds good.

Reply:
Take a look when you get a chance.